x-pack/plugins/stack_connectors/server/connector_types/stack/opsgenie/schema.ts

```diff
@@ -0,0 +1,130 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { schema } from '@kbn/config-schema';
+import { i18n } from '@kbn/i18n';
+
+export const ConfigSchema = schema.object({
+  apiUrl: schema.string(),
+});
+
+export const SecretsSchema = schema.object({
+  apiKey: schema.string(),
+});
+
+const responderTypes = schema.oneOf([
+  schema.literal('team'),
+  schema.literal('user'),
+  schema.literal('escalation'),
+  schema.literal('schedule'),
+]);
+
+const validateDetails = (details: Record<string, string>): string | void => {
```

Comment:
Sorry if I confused you before, but I meant that each value has to be less than 8K characters. Not collectively. You can have two values with 8K characters for each. 

<img width="1416" alt="Screenshot 2022-10-11 at 3 42 31 PM" src="https://user-images.githubusercontent.com/7871006/195094014-1163ba35-e8f3-495b-9ebe-a19879758217.png">

Reply:
Ooooooh got it.

Reply:
We decided to remove the validation offline because opsgenie will just truncate the strings and not throw an error.